### PR TITLE
Chronos: fix resample drop non-numeric columns

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

File: pyzoo/zoo/chronos/data/tsdataset.py [hunk 1]
```diff
@@ -288,6 +288,15 @@ def resample(self, interval, start_time=None, end_time=None, merge_mode="mean"):
         assert self._is_pd_datetime,\
             "The time series data does not have a Pandas datetime format\
             (you can use pandas.to_datetime to convert a string into a datetime format)."
+        from pandas.api.types import is_numeric_dtype
+        type_error_list = [val for val in self.target_col + self.feature_col
+                           if not is_numeric_dtype(val)]
```
Comment (TheaperDeng) [inline pyzoo/zoo/chronos/data/tsdataset.py:293]: `self.df[val]`

File: pyzoo/zoo/chronos/data/tsdataset.py [hunk 2]
```diff
@@ -288,6 +288,15 @@ def resample(self, interval, start_time=None, end_time=None, merge_mode="mean"):
...
+        try:
+            for val in type_error_list:
+                self.df[val] = self.df[val].astype(np.float32)
+        except Exception:
+            raise RuntimeError("All the columns of target_col \
```
Comment (TheaperDeng) [inline pyzoo/zoo/chronos/data/tsdataset.py:298]:
```python
RuntimeError("All the columns of target_col",
"and extra_feature_col should be of numeric type.")
```

**Review by TheaperDeng — APPROVED** [review]

**Comment by TheaperDeng** [comment]
pls test the speed of resample

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Validation/6395/

**Comment by liangs6212** [comment]
resample time increment <1s.
After converting the value to the object type, the resample time increment is ~4S, (test only)

**Pull request merged**